maps/nerva/nerva-2.dmm

```diff
@@ -13455,6 +13455,13 @@
 	pixel_y = -5;
 	req_access = list("ACCESS_BRIDGE")
 	},
+/obj/machinery/button/blast_door{
+	id_tag = "MainExpeditionPrep";
+	name = "Expedition Prep Blast Doors";
+	pixel_x = 5;
+	pixel_y = 5;
+	req_access = list("ACCESS_BRIDGE")
```

Comment:
should this not be expedition access

Reply:
It could be, but all the other buttons there are bridge access, it's in the bridge, everyone likely to press the button will have bridge access and maybe not prep access. I can change it if you think I should.

Reply:
Should probably be for consistency, the other buttons affect the bridge and this one doesn't. Plus not everyone with bridge access has expedition access, although most do.

Reply:
done